src/manage.c

```diff
@@ -4648,6 +4685,29 @@ run_osp_task (task_t task, int from, char **report_id)
   return 0;
 }
 
+/**
+ * @brief Get the number of retry on a scanner connection lost.
```

Comment:
This and the other comments below should say "number of retries" instead of "number of retry".